schema/common.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0 Library
+
+types:
+  PlainTextFastaResponse:
```

Comment:
i think probably it would be good to discuss the API by voice.  API design can be tricky, especially a complicated one like this